### DISCUSSION TITLE
Taskbar Volume Control mod doesn't work while scrolling using touchpad

### DISCUSSION THREAD

**Discussion by pota2dev** [ramensoftware/windhawk-mods#82] [Q&A]
Please add touchpad scrolling support in the mod.

**Comment by m417z** [comment]
I made some changes in version 1.1, can you see whether anything changed? I know that in Windows 10 it doesn't work, I added a notice about it in the readme, with a workaround to use the "pinch to zoom" gesture. But I'm not sure about Windows 11.
